docs/kickscooter_endpoint.md

```diff
@@ -219,12 +219,19 @@ curl --request GET \
 | **key** | **type** | required | **notes** |
 | --- | --- | --- | --- |
 | coverEndsAt | iso-8601 | Yes | string |
```

Comment:
@niveditazego can you also please update the coverEndsAt to be optional?

Reply:
done